electron/src/updater/windowsUpdater.ts

```diff
@@ -0,0 +1,125 @@
+"use strict";
```

Comment:
The updater implementation for Windows. Pretty much taken from VS Code. Downloads the new installer when told to, saves it to temp, then launches it as a detached process and exits the app. The installer takes care of overwriting all necessary files, rolling back in case of error, etc.